source/layouts/layout.erb

```diff
@@ -3,11 +3,13 @@
   <head>
     <meta charset="utf-8">
     <meta http-equiv="x-ua-compatible" content="ie=edge">
-    <title><%= current_page.data.title || "LearningSpaces" %></title>
-    <meta name="description" content="">
+    <title><%= current_page.data.title || "LearningSpaces" %><%= " | LearningSpaces" unless current_page.path == "index.html" %></title>
```

Comment:
so if there is a page without a title it'll be `LearningSpaces | LearningSpaces` ?